src/parts/Form/index.jsx

```diff
@@ -135,12 +136,26 @@ const Form = () => {
                         });
                     }
     
+                    const networkHead = head.header.height;
+                    const syncingWindowHeight = (30 * 24 * 60 * 60)/12;
+                    const syncingWindowTail = networkHead - syncingWindowHeight;
+                    //console.log("stored:", info.stored_headers);
+                    let storedRanges = info.stored_headers.map((range) => {
+                        const adjustedStart = Math.max(range.start-syncingWindowTail, 0);
+                        const adjustedEnd = Math.max(range.end-syncingWindowTail, 0);
+                        return { 
+                            start: adjustedStart/syncingWindowHeight,
+                            end: adjustedEnd/syncingWindowHeight
+                        };
+                    }).filter((range) => (range.end-range.start) > 0.01); // skip small <1% ranges
```

Comment:
can you pass there unmodified ranges and do this remap logic in progress bar directly? It would allow showing the current range too